src/main.h

```diff
@@ -5,6 +5,13 @@ namespace fallout {
 
 int falloutMain(int argc, char** argv);
 
+// True if game was started, false when on the main menu
+bool mainIsGameLoaded();
+void mainSetIsGameLoaded(bool isGameLoaded);
+
+bool mainIsInEndgameSlideshow();
+void mainSetIsInEndgameSlideshow(bool isInEndgameSlideshow);
```

Comment:
I suppose these could also live in `loop.h` as extra flags? Depends how much we want to stay close to sfall, I guess. WDYT?